config/core/300-resources/configuration.yaml

```diff
@@ -415,6 +415,13 @@ spec:
                                         items:
                                           description: Capability represent POSIX capabilities type
                                           type: string
+
+                                      add:
```

Comment:
We currently don't add feature flagged fields to the schema. Instead, you'll have to apply the `x-kubernetes-preserve-unknown-fields` tag to `securityContext`, so that the new field is transparently allowed.

Reply:
Done